composer.json

```diff
@@ -6,6 +6,7 @@
         "smarty/smarty" : "~3.1",
         "firebase/php-jwt" : "~3.0",
         "google/recaptcha": "~1.1",
+        "php-http/guzzle6-adapter": "*",
```

Comment:
Could you put something like ">=2.0"  (or at least >= 1.0) to require at least version 2 instead of any version?
I would not like unstable version as permitted by *